calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -189,6 +189,9 @@ For OpenShift, add the Felix configuration with WireGuard enabled [under custom
    > **Note**: The above command can be used to change other WireGuard attributes. For a list of other WireGuard parameters and configuration evaluation, see the [Felix configuration]({{site.baseurl}}/reference/resources/felixconfig#felix-configuration-definition).
    {: .alert .alert-info}
 
+   > **Note**:  `natOutgoing: true` is set for the default IPv4 IP pool, but not so for IPv6. Wireguard has issues in both IPv4 and IPv6 when `natOutgoing` is disabled, so [enable NAT outgoing for the IPv6 IP pools]({{site.baseurl}}/networking/workloads-outside-cluster) when using IPv6 Wireguard.
```

Comment:
Rather than "Wireguard has issues" we should say "Wireguard requires that NATOutgoing be enabled"

Reply:
Good point, thanks! Done.